### PR TITLE
bug 1717627. bump crv to 4.1.2

### PR DESCRIPTION
**Pull request by jcantrill**
This PR bumps CRV to 4.1.2 and adds art.yml file for future automation

### CODE REVIEW

File: manifests/4.1/elasticsearch-operator.v4.1.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ metadata:
     containerImage: quay.io/openshift/origin-elasticsearch-operator:latest
     createdAt: 2019-02-20T08:00:00Z
     support: AOS Cluster Logging, Jaeger
-    olm.skipRange: ">=4.1.0 <4.1.1"
+    olm.skipRange: ">=4.1.1 <4.1.2"
```
Comment (jwforres) [inline manifests/4.1/elasticsearch-operator.v4.1.0.clusterserviceversion.yaml]: 4.1.0

File: manifests/art.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+updates:
+  - file: "{MAJOR}.{MINOR}/elasticsearch-operator.v{MAJOR}.{MINOR}.0.clusterserviceversion.yaml" # relative to this file
+    update_list:
+    # replace metadata.name value
+    - search: "elasticsearch-operator.v{MAJOR}.{MINOR}.0"
```
Comment (jwforres) [inline manifests/art.yaml]: `.2` instead of `.0`

**Comment by jwforres** [comment]
/lgtm

approved for 4.1.2

**Comment by sudhaponnaganti** [comment]
@jcantrill - Can you pl reference bug number in the header. Is this related to 1717627?

**Pull request merged**